### PR TITLE
helm charts for oidc-gateway & etherpad

### PR DESCRIPTION
**Pull request by cmharlow**
Jira: https://mozilla-hub.atlassian.net/browse/SE-1151

What this PR does:
* Creates a Helm Chart for oidc-gateway (see https://github.com/mozilla-services/oidc-gateway), used by Etherpad but deployed manually using helm v1 template before
* Creates a Helm Chart for Etherpad, which expects in our implementation (but doesn't explicitly require as a dependency) oidc-gateway

### CODE REVIEW

File: charts/etherpad/templates/issuer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{{- if .Values.ingress.le.issuer_create }}
```
[the remark this answers is not in the document]
Comment (cmharlow) [reply charts/etherpad/templates/issuer.yaml:1]: I think the way we have cert-manager configured currently we do - it is namespace scoped (I believe to allow using either the Lets Encrypt stage or prod servers, though only Refractr Stage uses the LE staging servers currently).

If we want to change that behavior, we'd need to revisit cert-manager in the EKS terraform module & make its setup cluster-wide - then we can use I cluster-wide Issuer.

[the remark this answers is not in the document]
Comment (smarnach) [reply charts/etherpad/templates/issuer.yaml:1]: I don't think we need to change this – I was mainly curious.

File: charts/etherpad/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+maintainers:
+  - name: Web SRE Team
+    email: it-sre@mozilla.com
+
+dependencies:
+- name: cert-manager
+  version: v1.4.0
+  condition: cert-manager.install
+  repository: https://charts.jetstack.io
+- name: mysql
```
[the remark this answers is not in the document]
Comment (smarnach) [reply charts/etherpad/Chart.yaml:25]: I see, that makes sense.

**Comment by cmharlow** [comment]
Answered questions here - they're mostly things we need to put into our broader work backlog I believe, unless we want to alert & improve our helm charts CI & our cert-manager default EKS setup as part of this migration.

**Pull request merged**